From 9b69506ff9345e7c746cc9324642a5ff9634cd1a Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Wed, 10 Jul 2024 19:43:57 +0200
Subject: [PATCH] Remove sample entry point

---
 setup.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/setup.py b/setup.py
index d990352..2e14bae 100644
--- a/setup.py
+++ b/setup.py
@@ -62,9 +62,4 @@
     install_requires=['python-snappy', 'multitasking', 'toolz', 'partd',
                       'cloudpickle', 'distributed', 'pandas', 'numpy',
                       'pyarrow', 'dask[dataframe]'],
-    entry_points={
-        'console_scripts': [
-            'sample=sample:main',
-        ],
-    },
 )
